CHANGELOG.md

```diff
@@ -6,6 +6,9 @@
 
 ### Changes
 
+- Not related to development, but addded a releaser tool to help cut releases.
```

Comment:
@sneridagh do we want to include dev-related issues in the changelog? In plone.restapi I usually leave them out.

Reply:
Removed!